.rubocop_todo.yml

```diff
@@ -103,11 +103,12 @@ Lint/SymbolConversion:
   Exclude:
     - 'app/controllers/likes_controller.rb'
 
-# Offense count: 7
+# Offense count: 8
```

Comment:
This count goes up

Reply:
True. What happens often is one set of exclusions is exempting a whole file. Then it gets fixed, and three other problems are found.

For now, am focused on auto correcting the safe changes - will chip away - and then regenerating the todo file to reflect current state regularly.